### ISSUE TITLE
URGENT: Core MySQL PDO Error

### ISSUE THREAD

**Issue by j42** [facebook/hhvm#6700]
See: https://github.com/laravel/framework/issues/11537

**Comment by SiebelsTim** [comment]
It would be best if you could provide a self-contained test script to reproduce this issue.

**Comment by fredemmott** [comment]
- what code triggers the behavior?
 - what are you getting?
 - what are you expecting to get?

It is extremely unlikely that someone will investigate this without a stand-alone example (that doesn't depend on Laravel).

**Comment by j42** [comment]
Fair point, I was actually waiting to return to this until I heard a response from Taylor (re: Laravel).

Turns out it was their issue, he accepted my pull request, and I'm not sure if there ever was truly an issue with HHVM.  If it pops up again, I'll give you a full core dump with the offending code -- it's running on a production server with XX million daily actives so we'll see any anomalies fairly quickly.

**Comment by CreepGin** [comment]
@j42 Hi, I just ran into this exact issue. Disabling hhvm in homestead fixes the issue. But now you say it's not really an issue with HHVM. I searched through your pull requests and couldn't really find one that deals with PDO. How were you able to solve it?

**Comment by dschniepp** [comment]
@CreepGin did you resolved the issue?

**Comment by eliassorensen** [comment]
I experienced this issue too on on my development environment on AWS (EC2 with nginx+HHVM and RDS with MySQL 5.7).

Just a "General Error" with no clue whatsoever.

Installed PHP 7 and change the nginx configuration to use that, and with PHP 7 it just worked.

So it seems like it's an error with HHVM's PDO MySQL driver.
I also tried connecting to my Postgres database, but couldn't build the pgsql extension for HHVM on 3.13...

@j42 did you eventually resolve this issue?
